libs/wingc/src/ast.rs

```diff
@@ -202,6 +202,7 @@ pub enum StmtKind {
 	If {
 		condition: Expr,
 		statements: Scope,
+		elif_statements: Option<(Expr, Scope)>,
```

Comment:
```suggestion
		elifs_statements: Vec<(Expr, Scope)>,
```

Reply:
the above change makes a big impact here